### PR TITLE
feat: nextjs app directory example

### PR DESCRIPTION
**Pull request by DaniAkash**
## Motivation and Context

Add an example project on how to use WunderGraph with Next.js 13 App Directory along with both Server Components & Server Actions.

## Marketing Changelog

An Example TodoList app built with Next.js 13 App Directory along with Server Components & Server Actions.

#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jensneuse — APPROVED** [review]

**Comment by jensneuse** [comment]
This is super cool @DaniAkash!

**Review by StarpTech — CHANGES_REQUESTED** [review]
Please add an integration test per example.

**Review by StarpTech — APPROVED** [review]
LGTM

File: examples/nextjs-relay-todos/.wundergraph/operations/relay/persisted.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "2f7a7ed13c67c406b6adaacf8aea2780": "query pagesAllTodosQuery {\n  todos_todos {\n    ...TodoList_todo\n  }\n}\n\nfragment TodoList_todo on todos_Todo {\n  id\n  text\n  isCompleted\n}\n",
   "3ece2b70448e57683ea10290c6a68b8b": "subscription pagesOnTodoChangesSubscription {\n  todos_TodoChanges {\n    todoID: id\n    text\n    isCompleted\n  }\n}\n",
+  "5ce551f3c418699ad61443ed34aa470f": "mutation TodoListAddTodoMutation(\n  $todo: todos_NewTodoInput!\n) {\n  todos_addTodo(todo: $todo) {\n    id\n    text\n    isCompleted\n  }\n}\n",
```
Comment (Balance8) [inline examples/nextjs-relay-todos/.wundergraph/operations/relay/persisted.json:4]: Why does the relay app need to be updated with this example?

Comment (DaniAkash) [reply examples/nextjs-relay-todos/.wundergraph/operations/relay/persisted.json]: Seems like an accidental change - reverting it

**Pull request merged**